samples/system-test/quickstart.test.js

```diff
@@ -28,30 +28,29 @@ const expectedDatasetId = `my_new_dataset`;
 let datasetId = `nodejs-docs-samples-test-${uuid.v4()}`;
 datasetId = datasetId.replace(/-/gi, `_`);
 
-test.after.always(async () => {
-  try {
-    bigquery.dataset(datasetId).delete({force: true});
-  } catch (err) {} // ignore error
-});
-
-test.beforeEach(tools.stubConsole);
-test.afterEach.always(tools.restoreConsole);
+describe(`Quickstart`, () => {
+  beforeEach(tools.stubConsole);
```

Comment:
Feels like maybe these belong towards the top in the outer describe?

Reply:
No it's placed under describe everywhere